ingestion-data/collections/CMIP245-winter-median-pr.json

```diff
@@ -39,5 +39,18 @@
     },
     "dashboard:is_periodic": false,
     "dashboard:time_density": "year",
-    "stac_version": "1.0.0"
+    "stac_version": "1.0.0",
+    "renders": {
+        "dashboard": {
+            "resampling": "bilinear",
+            "bidx": 1,
+            "nodata": "nan",
+            "colormap_name": "rdbu",
+            "rescale": [
+                -60,
+                60
+            ],
+            "title": "VEDA Dashboard Render Parameters"
+        }
+    }
```

Comment:
I think we are missing the `assets` key

Reply:
I don't think we currently use them for the dashboard but I can add

Reply:
I think `assets` is the only required key of the render extension

Reply:
Oh right I understand it now, just read the spec again. I'll add it in there.